devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -53,8 +56,20 @@ metadata {
 }
 
 def initialize() {
-	// Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
-	sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	if (zwaveInfo.mfr.equals("0086"))
+		// 4 hour ping for Aeotec
+		sendEvent(name: "checkInterval", value: 4 * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	else
+		// 12 hours for other devices
+		sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def installed() {
+	initialize()
+	//water alarm
+	def cmds = [ encap(zwave.notificationV3.notificationGet(notificationType: 0x05)),
+				 encap(zwave.batteryV1.batteryGet())]
+	response(cmds)
 }
 
 def installed() {
```

Comment:
looks like you have two `installed()` methods now.

Reply:
Done.